[error] scala.ScalaReflectionException: object org.scalatest.prop.Configuration$ not found.
[error] 	at scala.reflect.internal.Mirrors$RootsBase.staticModule(Mirrors.scala:162)
[error] 	at scala.reflect.internal.Mirrors$RootsBase.staticModule(Mirrors.scala:22)
[error] 	at org.scalatest.tools.Framework.runner(Framework.scala:1021)
[error] 	at sbt.Defaults$.$anonfun$createTestRunners$1(Defaults.scala:944)
[error] 	at scala.collection.TraversableLike.$anonfun$map$1(TraversableLike.scala:233)
[error] 	at scala.collection.immutable.Map$Map1.foreach(Map.scala:125)
[error] 	at scala.collection.TraversableLike.map(TraversableLike.scala:233)
[error] 	at scala.collection.TraversableLike.map$(TraversableLike.scala:226)
[error] 	at scala.collection.AbstractTraversable.map(Traversable.scala:104)
[error] 	at sbt.Defaults$.createTestRunners(Defaults.scala:938)
[error] 	at sbt.Defaults$.allTestGroupsTask(Defaults.scala:1000)
[error] 	at sbt.Defaults$.$anonfun$testTasks$10(Defaults.scala:756)
[error] 	at scala.Function1.$anonfun$compose$1(Function1.scala:44)
[error] 	at sbt.internal.util.$tilde$greater.$anonfun$$u2219$1(TypeFunctions.scala:40)
[error] 	at sbt.std.Transform$$anon$4.work(System.scala:67)
[error] 	at sbt.Execute.$anonfun$submit$2(Execute.scala:269)
[error] 	at sbt.internal.util.ErrorHandling$.wideConvert(ErrorHandling.scala:16)
[error] 	at sbt.Execute.work(Execute.scala:278)
[error] 	at sbt.Execute.$anonfun$submit$1(Execute.scala:269)
[error] 	at sbt.ConcurrentRestrictions$$anon$4.$anonfun$submitValid$1(ConcurrentRestrictions.scala:178)
[error] 	at sbt.CompletionService$$anon$2.call(CompletionService.scala:37)
[error] 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
[error] 	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
[error] 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
[error] 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
[error] 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
[error] 	at java.lang.Thread.run(Thread.java:745)
[error] (Test / executeTests) scala.ScalaReflectionException: object org.scalatest.prop.Configuration$ not found.
